src/clojure_lsp/crawler.clj

```diff
@@ -61,8 +61,8 @@
                               (enumeration-seq)
                               (remove #(.isDirectory %))
                               (map (fn [entry]
-                                     [(if jar-dependency-scheme?
-                                        (str "jar:file://" jar-file "!/" (.getName entry))
+                                     [(if (= "jar" dependency-scheme)
```

Comment:
Why we need this change?

Reply:
Oops, must have merged poorly. I meant to accept the new `jar-dependency-scheme?`. I'll fix.